user/tests/wiring/api/watchdog.cpp

```diff
@@ -0,0 +1,57 @@
+
+#include "testapi.h"
```

Comment:
I'll add a small feature to test runner next week so that we can expect a reset from within the tests and we can add an automated test for the watchdog as well.